smithy-aws-traits/src/test/resources/software/amazon/smithy/aws/traits/errorfiles/protocols/aws-query-compatible-does-not-support-non-aws-json.smithy

```diff
@@ -0,0 +1,23 @@
+$version: "2.0"
+
+namespace smithy.example
+
+use aws.protocols#awsQueryCompatible
+use aws.protocols#awsQueryError
+use aws.protocols#awsQuery
+
+@awsQueryCompatible
+@awsQuery
```

Comment:
Can you add a fake `xmlNamespace` to this service to clean up that error in the test listing?